### PR TITLE
Type declarations for BreeJS

### PR DESCRIPTION
**Pull request by shadowgate15**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by shadowgate15** [comment]
I am a contributor to BreeJS.

File: types/bree/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
+// Type definitions for bree 6.2
+// Project: https://github.com/breejs/bree
+// Definitions by: Taylor Schley <https://github.com/shadogate15>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ This declaration specifies that the class constructor function
+ *~ is the exported object from the file
+ */
+
```
Comment (peterblazejewicz) [inline types/bree/index.d.ts:9]:
```suggestion
```
these are instruction from the template for the writer

Comment (shadowgate15) [reply types/bree/index.d.ts]: removed

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@shadowgate15 thx!

**Comment by shadowgate15** [comment]
Ready to merge

**Pull request merged**